src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const hasPartner = () => {
+    if (!isMarried) {
+      return 'I am not married';
+    }
+
+    return sex === 'm'
+      ? `${partnerName} is my wife`
+      : `${partnerName} is my husband`;
+  };
+
+  const renderAge = () => (
+    <p className="Person__age">
+      {`I am ${age}`}
+    </p>
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {person.age && renderAge()}
+
+      <p className="Person__partner">
+        {hasPartner()}
```

Comment:
nice usage of helpers function for conditional rendering of maried status but name of function can be more declarating. Like "getMariedStatusMessage" or something like that)